SPECS/ocaml-findlib/ocaml-findlib.spec

```
@@ -64,39 +85,30 @@ make install \
      prefix=%{buildroot} \
      OCAMLFIND_BIN=%{_bindir} \
      OCAMLFIND_MAN=%{_mandir}
+ 
+# Remove spurious executable bits
+chmod 0644 %{buildroot}%{_mandir}/man{1,5}/*
+chmod 0644 %{buildroot}%{_libdir}/ocaml/findlib/*.{cma,cmi,ml,mli}
+chmod 0644 %{buildroot}%{_libdir}/ocaml/findlib/{META,Makefile*}
+%ifarch %{ocaml_native_compiler}
+chmod 0644 %{buildroot}%{_libdir}/ocaml/findlib/*.{a,cmxa}
+%endif
 
+%ocaml_files
+sed -i '/ocamlfind\.conf/d' .ofiles
 
-%files
-%license LICENSE
-%doc doc/README
+%files -f .ofiles
+%doc LICENSE doc/README
```

No, license and documentation need to be separate. Installing a package without the documentation should still install the license. That's the main reason licenses cannot be marked as documentation.
```suggestion
%license LICENSE
%doc doc/README
```